site/content/documentation/reference/rest-api.md

```diff
@@ -155,17 +155,13 @@ Response:
     <?xml version='1.0' encoding='UTF-8'?>
     <sparql xmlns='http://www.w3.org/2005/sparql-results#'>
       <head>
-             <variable name='&lt;foo:bar&gt;'/>
       </head>
-      <results ordered='false' distinct='false'>
-             <result>
-                    <binding name='&lt;foo:bar&gt;'>
-                      <uri>foo:bar</uri>
-                    </binding>
-             </result>
+      <results>
       </results>
     </sparql>
 
+(note that in this example the query result is empty)
```

Comment:
:D

Reply:
I was struggling with finding a decent reproducible example that didn't result in a very long encoded query. This is the best I could do :) There's more comprehensive examples further down though.

Reply:
I assumed something along the lines, and anyone reading this documentation will be looking for the actual request more than the returned results. Just thought it was a cute little workaround and actually fairly representative since empty results are also valid responses.